### PR TITLE
Added subclasses of morphology based neurons

### PR DESCRIPTION
**Pull request by shawntanzk**

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
 SubClassOf(obo:CL_4023081 obo:CL_4023050)
 SubClassOf(obo:CL_4023081 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070021))
 
+# Class: obo:CL_4023083 (chandelier cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15378039") obo:IAO_0000115 obo:CL_4023083 "A type of interneuron that are axon-targeting. They can be multipolar or bitufted. Their local axonal clusters are formed by high-frequency branching at shallow angles, often ramifying around, above or below their somata with a high bouton density. The characteristic terminal portions of the axon form short vertical rows of boutons, resembling a chandelier.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28262]: "A type of" is redundant for classes. Should be "An interneuron that is axon-targeting." 

Also, is any interneuron that is axon-targeting a chandelier cell? I thought that ChCs synapsed to axons of pyramidal cells. Perhaps that information is out-of-scope for now.

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28262]: that's a good point - will try to reorder it a bit so the main part comes first. They definitely do synapse to pyramidal cells - should have been worked it, sorry about that. 

Thinking this instead:

An interneuron that selectively innervates the axon initial segment of pyramidal cells. Their local axonal clusters are formed by high-frequency branching at shallow angles, often ramifying around, above or below their somata with a high bouton density. The characteristic terminal portions of the axon form short vertical rows of boutons, resembling a chandelier. Chandelier cells can be multipolar or bitufted.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
...
+
+# Class: obo:CL_4023086 ('T-shaped' Martinotti neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/b978-0-12-369497-3.10004-4"^^xsd:string) obo:IAO_0000115 obo:CL_4023086 "A Martinotti neuron that has axons that form a horizontal ramification, making it T-shaped.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023086 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:inSubset obo:CL_4023086 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
+AnnotationAssertion(rdfs:label obo:CL_4023086 "'T-shaped' Martinotti neuron")
+EquivalentClasses(obo:CL_4023086 ObjectIntersectionOf(obo:CL_4023076 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070008)))
+
+# Class: obo:CL_4023087 ('fanning-out' Martinotti neuron)
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28277]: This is really nit-picky, but why does the label have shudder quotes? The hyphenated gerund+preposition is a bit weird to me as well. I suppose that is historical precedent, though.

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28277]: yeah it is a PAIN to add the PATO term as an axiom - protege really does not like have two ' consecutively and I have been doing a work around by removing it, then adding it back in after I have placed the axiom. I really would like the remove it, but at the moment I'm directly copying it from the paper. Thoughts on this @dosumis?

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
...
+# Class: obo:CL_4023092 (inverted pyramidal neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30715238") obo:IAO_0000115 obo:CL_4023092 "A pyramidal neuron which has an apical tree which is oriented towards the white matter.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023092 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023092 "inverted pyramidal neuron")
+EquivalentClasses(obo:CL_4023092 ObjectIntersectionOf(obo:CL_0000598 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070021)))
+
+# Class: obo:CL_4023093 (stellate pyramidal neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30715238") obo:IAO_0000115 obo:CL_4023093 "A pyramidal neuron which lack a tuft formation but extends to small radial distances forming a star-like shape.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28318]: 'lack' -> 'lacks'

'extends to' -> 'extends'

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023093 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023093 "stellate pyramidal neuron")
+EquivalentClasses(obo:CL_4023093 ObjectIntersectionOf(obo:CL_0000598 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070020)))
+
+# Class: obo:CL_4023094 (tufted pyramidal neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30715238") obo:IAO_0000115 obo:CL_4023094 "A pyramidal neuron which has a distinct tuft formation, distal from the soma.")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:28325]: 'distinct' -> 'distinctive'

**Review by patrick-lloyd-ray — APPROVED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (patrick-lloyd-ray) [reply src/ontology/cl-edit.owl:28262]: Awesome!

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (patrick-lloyd-ray) [reply src/ontology/cl-edit.owl:28277]: Ah, no problem then.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl:28262]: Yep - second version *much* better.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl:28277]: Please lose the shudder quotes.  If you want to keep them for search purposes, add a synonym.

Comment (dosumis) [reply src/ontology/cl-edit.owl:28277]: Maybe 'fan Martinotti neuron' for the label - if you can find precedent.   Not a big deal though.

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28277]: will remove them, and will update PATO accordingly too. Thanks

**Pull request merged**